cfgov/v1/templatetags/share.py

```diff
@@ -42,3 +42,8 @@ def v1page_permissions(context, page):
         )
 
     return context['user_page_permissions'].for_page(page)
+
+@register.filter
+def get_raw_html(url):
+    r = requests.get(url)
+    return r.content
```

Comment:
This will return the content of the request even if it fails, e.g. a 500 error message. It might be worth checking the status code or using [`r.raise_for_status`](http://docs.python-requests.org/en/master/user/quickstart/#response-status-codes) to raise an exception if the request fails. This depends somewhat on the desired behavior should the request fail; should this cause the entire page view to fail, or substitute some kind of error text instead? My preference would be for the former.